static/bidder-params/richaudience.json

```diff
@@ -0,0 +1,31 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "richaudience Adapter Params",
+	"description": "A schema which validates params accepted by the richaudience adapter",
+	"type": "object",
+
+	"properties": {
+		"pid": {
+		  "type": "string",
+		  "description": "Placement ID"
+		},
+		"supplyType": {
+			"type": "string",
+			"description": "Type integration"
+		},
+		"testRa": {
+			"type": "boolean",
+			"description": "TestMode"
+		},
+		"BidFloor": {
+			"type": "number",
+			"description": "floor price"
+		},
+		"BidFloorCur": {
+			"type": "string",
+			"description": "currency floor price"
+		}
```

Comment:
Please consider using consistent capitalization. `BidFloor` and `BidFloorCur` start with an upper case letter but `pid`, `supplyType`, and `testRa` do not.

Reply:
in next commit i have solved